conans/client/remote_manager.py

```diff
@@ -147,6 +155,7 @@ def get_package(self, pref, dest_folder, remote, output, recorder):
                 metadata.packages[pref.id].revision = pref.revision
                 metadata.packages[pref.id].recipe_revision = pref.ref.revision
                 metadata.packages[pref.id].checksums = package_checksums
+                metadata.packages[pref.id].remote = remote.name
```

Comment:
This will assign the remote to ``conan download`` binaries. So far it seems it was not assigned, which seems a bit inconsistent, as the defined behavior is that packages keep the remote from which it was installed from.

Reply:
👍 

It was assigned for the `conan install`, but not for the `conan download`, bug or feature? 🤔  I prefer to be consistent.

Reply:
To be honest, this was a surprise to me. That you did a ``conan download`` and the remote was not assigned, I observed it as a buggy behavior. Even more if there is a chance that such binary (maybe a previous revision) came from another remote, and maintains that remote. So I think we should be consistent and assign the remote: bug.